environment.yml

```diff
@@ -9,9 +9,11 @@ dependencies:
     - fastqc>=0.11
     - gzip>=1.7
     - mash>=2.3
+    - megahit>=1.2
     - pytest-cov>=3.0
     - quast>=5.0
     - seqtk>=1.3
     - snakemake>=6.10
     - spades>=3.15
+    - unicycler>=0.5
```

Comment:
I get the following when I try to set up a yeat environment from scratch on macOS.

```
$ mamba env update -f environment.yml 
conda-forge/osx-64                                          Using cache
conda-forge/noarch                                          Using cache
bioconda/osx-64                                             Using cache
bioconda/noarch                                             Using cache
pkgs/main/osx-64                                              No change
pkgs/main/noarch                                              No change
pkgs/r/noarch                                                 No change
pkgs/r/osx-64                                                 No change


Looking for: ['black=21.10b0', "fastp[version='>=0.23']", "fastqc[version='>=0.11']", "gzip[version='>=1.7']", "mash[version='>=2.3']", "megahit[version='>=1.2']", "pytest-cov[version='>=3.0']", "quast[version='>=5.0']", "seqtk[version='>=1.3']", "snakemake[version='>=6.10']", "spades[version='>=3.15']", "unicycler[version='>=0.5']", "versioneer[version='>=0.21']"]


Encountered problems while solving:
  - nothing provides requested unicycler >=0.5
```

Have you only tested on WSL, did you use an earlier version on macOS, did you install from source, or something else?

Reply:
Sounds good. I have dropped the `unicycler` version number to 0.4 and is working now.